### PR TITLE
Simplify add event implementation

### PR DESCRIPTION
**Pull request by kieranyyu**
About:
This is a modification of adding an event to directly call the event servlet.

Changes:
The POST method in EventServlet.java has been simplified.
In add_event_page/script.js, the form is submitted directly with the form action and hidden input fields send data not directly recorded by the form.

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/EventServlet.java [hunk 1]
```diff
@@ -76,56 +90,37 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
-      response.getWriter().println("You've called the get request");
+      Long event_id = Long.parseLong(request.getParameter("event_id"));
+      KeyFactory keyFactory = datastore.newKeyFactory().setKind("Event");
+      Entity entity = datastore.get(keyFactory.newKey(event_id));
+      Event event = createEventFromEntity(entity);
+
+      Gson gson = new Gson();
+      response.setContentType("application/json;");
+      response.getWriter().println(gson.toJson(event));
+      System.out.println(gson.toJson(event));
```
Comment (marcusblake) [inline src/main/java/com/google/sps/servlets/EventServlet.java]: nit: remove system out print

File: src/main/java/com/google/sps/servlets/EventServlet.java [hunk 2]
```diff
@@ -76,56 +90,37 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
+    String event_type = request.getParameter("event_type");
+    String event_sub = request.getParameter("subject");
+    Long event_school_L = Long.parseLong(request.getParameter("school_id"));
 
     // Store to the Datastore
     Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
     KeyFactory keyFactory = datastore.newKeyFactory().setKind("Event");
     LatLng position = LatLng.of(event_Lat_D, event_Lng_D);
-    FullEntity eventEntity =
+    FullEntity<IncompleteKey> eventEntity =
```
Comment (marcusblake) [inline src/main/java/com/google/sps/servlets/EventServlet.java:122]: out of curiosity: what's the benefit of adding `IncompleteKey`?

File: src/main/java/com/google/sps/servlets/EventServlet.java (hunk 1 above)
Comment (kieranyyu) [reply src/main/java/com/google/sps/servlets/EventServlet.java]: Done! Also, I copy-pasted this from Seokha's pull request just to see how the entire file would be laid out, wouldn't want to take credit for it myself.

File: src/main/java/com/google/sps/servlets/EventServlet.java (hunk 2 above)
Comment (kieranyyu) [reply src/main/java/com/google/sps/servlets/EventServlet.java:122]: This is to get rid of a warning that says FullEntity is a raw type. IncompleteKey specifies that the entity is created without an id given and datastore will generate it automatically.

File: src/main/java/com/google/sps/servlets/EventServlet.java (hunk 1 above)
Comment (marcusblake) [reply src/main/java/com/google/sps/servlets/EventServlet.java]: It is okay for now, but for future reference, wait a bit for the PR owner to submit their code (maybe unless it's an emergency and they aren't able to submit to resolve it).

**Review by marcusblake — APPROVED** [review]

**Comment by marcusblake** [comment]
Make sure to close the other PR that has these changes

**Review by mingyi09 — APPROVED** [review]

**Pull request merged**